arrays.ts

```diff
@@ -298,4 +298,5 @@ export const SUBJECTS: Entry[] = [
   "guv'nor",
   'captain',
   'princess',
+  { word: 'bottom', cw: 'kinky' },
```

Comment:
I guess it nicely avoids the potential for `cw: lewd, lewd` but I don't really see `bottom` as "kinky" per se 🤔

Reply:
uh fair... perhaps D/s would be better?